KOA/clanwars/apps.py

```diff
@@ -1,5 +1,5 @@
 from django.apps import AppConfig
-
+import json
```

Comment:
Is this import useful?

Reply:
yes it allows for use of the json() function to parse data as json so it can be read